editing/fx/passes/pact/harmonize.py

```diff
@@ -509,12 +497,11 @@ def disassemble_layernorm_fun(gm : fx.GraphModule, match : Match):
 
     return torch.nn.Sequential(*[new_layernorm, batchnorm, activation])
 
-
 class LayerNormDisassemblePass(SequentialPass):
-    def __init__(self, **kwargs):
+    def __init__(self, symbolic_trace: callable = PACT_symbolic_trace, **kwargs):
```

Comment:
The type annotation looks wrong. I think it should be Callable[[Union[nn.Module, fx.GraphModule]], fx.GraphModule] or similar.

Reply:
Addressed in [this commit](https://github.com/pulp-platform/quantlib/pull/15/commits/c2553bef1a29dc0baf6dd19cf1cbd8a2a351cedf)